CIS Scripts/1_Set_Organization_Priorities.sh

```diff
@@ -131,7 +131,7 @@ OrgScore2_4_4="true"
 
 # 2.4.5 Disable Remote Login 
 # SSH
-OrgScore2_4_5="true"
+OrgScore2_4_5="false"
```

Comment:
Is there a reason for this? I.e. remote administration?

Reply:
Yep, precisely.

Reply:
Fair enough!